### PR TITLE
Homework for Lesson 3

### PR DESCRIPTION
**Pull request by filkodaria**
Homework_3: task_1, task_2, task_3.

### CODE REVIEW

File: hw_3/hw_3.3.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+*/
+
+let n = 9;
+// let n = 15;
+// let n = 1.5;
+// let n = 'a';
+
+let countSum = n + Number(String(n) + n) + Number(String(n) + n + n);
+
+if (Number.isInteger(n) && n >=1 && n <=9) {
```
Comment (anatoly-karpovich) [inline hw_3/hw_3.3.js:11]: Проверку на число лучше делать перед вычислениями

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]
Please read the comment and apply changes

File: hw_3/hw_3.3.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+n + nn + nnn, где n не перемножаются, а конкатенируются
+*/
+
+let n = 9;
+// let n = 15;
+// let n = 1.5;
+// let n = 'a';
+
+let countSum = n + Number(String(n) + n) + Number(String(n) + n + n);
```
Comment (anatoly-karpovich) [inline hw_3/hw_3.3.js]: Провериь файлы с task2 и task3 на предмет let / const. Если переменная после создания не меняется - следует использовать const

File: hw_3/hw_3.3.js (hunk 1 above)
Comment (filkodaria) [reply hw_3/hw_3.3.js:11]: done

File: hw_3/hw_3.3.js (hunk 2 above)
Comment (filkodaria) [reply hw_3/hw_3.3.js]: done

**Comment by filkodaria** [comment]
I've added changes to the files hw_3.2.js and hw_3.3.js

**Review by anatoly-karpovich — APPROVED** [review]

File: hw_3/hw_3.2.js [hunk 3]
```diff
@@ -6,25 +6,23 @@ Task2*. Решить квадратные уравнения. Переменны
...
 
-let a1 = 1;
-let b1 = -6;
-let c1 = 9;
+const a1 = 1;
+const b1 = -6;
+const c1 = 9;
 
-let discriminant1 = b1 ** 2 - 4 * a1 * c1;
+const discriminant1 = b1 ** 2 - 4 * a1 * c1;
```
Comment (VladimirLQA) [inline hw_3/hw_3.2.js:14]: Or you could use `let`, but reasign them and reuse 
`const discriminant = b ** 2 - 4 * a * c;` for second equation

File: hw_3/hw_3.2.js [hunk 4]
```diff
@@ -6,25 +6,23 @@ Task2*. Решить квадратные уравнения. Переменны
...
+
+const x1 = (-b1 + Math.sqrt(discriminant1)) / (2 * a1);
+const x2 = (-b1 - Math.sqrt(discriminant1)) / (2 * a1);
```
Comment (VladimirLQA) [inline hw_3/hw_3.2.js:17]: Looks like it's possible to improve the naming for this variables.
Provided formula is used to find the roots of a quadratic equation.
As an options: `root` or `quadraticRoot`

Comment (filkodaria) [reply hw_3/hw_3.2.js:17]: thanks for your advices

**Pull request merged**